website_sale_stock_available_display/models/product_template.py

```diff
@@ -11,3 +13,36 @@ class ProductTemplate(models.Model):
         ('always_no_lock',
          'Show inventory on website and allow sales if not enough stock'),
     ])
+
+    @api.model
+    def fields_view_get(self, view_id=None, view_type='form', toolbar=False,
+                        submenu=False):
+        """The purpose of this is to write a invisible key on field attributes
+         respecting other invisible keys on this field.
+         There is a PR (https://github.com/odoo/odoo/pull/26607) to odoo for
+         avoiding this. If merged, remove this method and add the attribute
+         in the field.
+         """
+        res = super().fields_view_get(
+            view_id=view_id, view_type=view_type, toolbar=toolbar,
+            submenu=submenu,
+        )
+        if view_type == 'form':
+            view_xml = etree.XML(res['arch'])
+            xml_field = view_xml.xpath("//field[@name='custom_message']")
+            if xml_field:
+                field = xml_field[0]
+                attribs = safe_eval(field.attrib.get("attrs", "{}"))
```

Comment:
Don't `safe_eval` this as it can contain unevaluable things (like a field name). You have to inject it through strings:

https://github.com/OCA/commission/blob/05a05f3060c2085f014ec231296e51659e675b59/sale_commission/models/sale_order.py#L28-L48